torchaudio/io/_stream_writer.py

```diff
@@ -0,0 +1,257 @@
+from typing import Dict, Optional
+
+import torch
+
+
+def _format_doc(**kwargs):
+    def decorator(obj):
+        obj.__doc__ = obj.__doc__.format(**kwargs)
+        return obj
+
+    return decorator
+
+
+_encoder = """The name of the encoder to be used.
+                When provided, use the specified encoder instead of the default one.
+
```

Comment:
nit: the additional blank lines here cause the following line to be output on a new line. I think they can all be put within the same block, without needing the command/default to be in separate lines.